### DISCUSSION TITLE
不想让标题换行（或者希望标题手动换行后页眉可以保持单行）

### DISCUSSION THREAD

**Discussion by xiling1204** [sjtug/SJTUThesis#803] [提问] [answered]
bachelor thesis，论文标题只长了一点点，自动换行卡在最后一个字那里。尝试过\\\手动换行，但是发现页眉的标题也会变成两行。
怎样才能让标题不换行，或手动换行后页眉可以保持单行呢？
（现在能想到的做法只有把换行了的封面重新生成一个pdf之后，和原先的剩余部分拼接

**Comment by skyzh** [comment, marked as answer]
https://github.com/sjtug/SJTUThesis/blob/b164d0f5b3087ed86c1038b5a85014b47a92c6f2/setup.tex#L17-L29

setup.tex 里面区分了页眉标题和封面标题。

**Reply by xiling1204** [reply]
感谢！没发现注释里有
